packages/amplify_api/lib/src/method_channel_api.dart

```diff
@@ -251,6 +251,9 @@ class AmplifyAPIMethodChannel extends AmplifyAPI {
     inputsMap['restOptions'] = restOptions.serializeAsMap();
     inputsMap['cancelToken'] = cancelToken;
 
+    // Refresh auth tokens
+    await _authProviderRefresher();
```

Comment:
This change has been reverted on main, which should not be brought into RC. Please remove corresponding commit.

Reply:
Ah thank you.  Reverted the corresponding commit.